packages/neo-one-build-tests/jest/unit.js

```diff
@@ -3,23 +3,14 @@ const base = require('./base');
 module.exports = {
   ...base({ path: 'test' }),
   displayName: 'unit',
+  moduleNameMapper: {
```

Comment:
I'm sure this relates to the answer to my comment on `e2e.js` above, but curious why we didn't have this here before?